File: src/my_project/easy_problems/from151To200/reverse_string.py
from typing import List 

class Solution:
    def reverseString(self, s: List[str]) -> None:
        len_s: int = len(s)
        for i in range(len_s//2):
            s[i], s[len_s-1-i] =  s[len_s-1-i] , s[i]
        return s 


solution = Solution()

print(solution.reverseString(s = ["h","e","l","l","o"]))

print(solution.reverseString(s = ["H","a","n","n","a","h"]))

File: my_project_typescript/easy_problems_typescript/from51To100/reverseString.ts
function reverseString(s: string[]): void {

    let lenS: number = s.length
    let tempLeft: string
    let tempRight: string
    for (let i = 0; i < Math.floor(lenS/2); i++){
        tempLeft = s[i]
        tempRight = s[lenS-1-i]
        s[i] = tempRight
        s[lenS-1-i] = tempLeft
    }
    

};


reverseString(["h","e","l","l","o"])

reverseString(["H","a","n","n","a","h"])


